hail/src/main/scala/is/hail/expr/types/MatrixType.scala

```diff
@@ -135,6 +135,8 @@ case class MatrixType(
     sb.append(s"entry:$space")
     entryType.pretty(sb, indent, compact)
 
+    sb.append(s"entriesIndex:$space${entriesIdx}")
```

Comment:
Where else is this used? Is it parsable or only for debugging?

Reply:
@cseed ?

I assume `pretty` is not for parseable things, just for human-readable things.

Reply:
on BaseType:
```
  def parsableString(): String = toPrettyString(0, compact = true)
```

Reply:
🤷‍♀️

Reply:
Uh, is it bad that we don't preserve entries location across a write/read? @tpoterba

Reply:
Rows and entries get split during a write, so they have to get joined back together during a read anyways.